server.js

```diff
@@ -23,8 +15,59 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('', (req, res) => {
+  res.end('Learn about avocado sales. Jeeiij')
+})
+
+// /sales - all sales everywhere
+// /sales?totalBagsSoldMoreThan=1234 - all sales with more than 1234 sold bags
+app.get('/sales', (req, res) => {
+  const totalBagsSoldMoreThan = req.query.totalBagsSoldMoreThan
+  if (totalBagsSoldMoreThan) {
+    const matchingSales = sales.filter(
+      (sale) => sale.totalBagsSold >= totalBagsSoldMoreThan
+    )
+    res.json(matchingSales)
+  } else {
+    res.json(sales)
+  }
+})
+
+// average price, sorted from low to high
+app.get('/salesByAveragePrice', (req, res) => {
+  res.json(sales.sort((a, b) => a.averagePrice - b.averagePrice))
+})
+//combine this route with the sales route if there is time
+
+// /sales/5 - sale with id 5 in every region
+app.get('/sales/:id', (req, res) => {
+  const id = req.params.id
+  const saleIdInAllRegions = sales.filter((sale) => sale.id === +id)
+  res.json(saleIdInAllRegions)
+})
+
+// /Albany/sales - all sales in Albany
+app.get('/:region/sales', (req, res) => {
```

Comment:
Nice! This could also be a filter on the `/sales` endpoint but having the region in front makes it still feel RESTful so I'm not going to complain ;)